Airport2/Airport2/Passenger.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Airport2.Enums;
+
+namespace Airport2
+{
+    public class Passenger
+    {
+        public readonly Guid Id;
+
+        public Guid FlightNumber { get => this.Flight == null ? Guid.Empty : this.Flight.Number; }
+
+        public string FirstName { get; private set; }
+        public string SecondName { get; private set; }
+        public string Nationality { get; private set; }
+        public DateTime Birthday { get; private set; }
+        public Gender Gender { get; private set; }
+        public Flight Flight { get; private set; }
```

Comment:
You shure that passenger can have only one ticket?  What if he wants to get ticket to there and back?